qbertconfig/Fetcher.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2018 Platform9 Systems, Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy
+# of the License at
+
+#   http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+import logging
+import openstack
+import sys
+from keystoneauth1.exceptions import MissingRequiredOptions
+
+# Local imports
+from QbertClient import QbertClient
+from Kubeconfig import Kubeconfig
+
+LOG = logging.getLogger(__name__)
+
+
+class Fetcher(object):
+    """ Fetches a Kubeconfig from Qbert """
+    def __init__(self, kubeconfig=None, os_config=None, os_cloud=None):
+        """
+        :param kubeconfig: qbertconfig.Kubeconfig object
```

Comment:
Minor: Can you use the same doc styles as other functions? I was going with google's style guidelines found here: https://github.com/google/styleguide/blob/gh-pages/pyguide.md#383-functions-and-methods